doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+1. __Refactor components into connected components.__ Update the component to `UiComponent2`, removing all Flux boilerplate, and wrapping the already declared factory with the `connect` call.
+1. __Move on to the next group.__ If the refactor has multiple groups of stores and components, move on to the next one.
+1. __Add a `ReduxProvider` around the component tree.__ Once it makes sense, wrap the tree in the `ReduxProvider` component.
+
+### What if the Advanced Conversion is too Daunting?
+The advanced conversion might sound like a huge initiative with numerous unknowns and complexities. If that's the case the first question is whether Redux will offer the benefits to justify a large initiative. Our [OverReact Redux documentation](./over_react_redux_documentation.md) includes some of the benefits, and there are many [articles online](https://www.google.com/search?q=the+benefits+of+redux&oq=the+benefits+of+redux) that provide more examples. 
+
+If Redux makes sense but the conversion project seems extremely challenging, we have a middleground: introducing __Influx__.
+
+## Influx Architecture
+### What is Influx?
+Influx is just the term we're giving to an architecture that is both Redux and Flux at the same time. To aid in the transition from Flux to Redux, tools have been built to allow Flux and Connected (Redux) components to all talk to the same store. It's a transitional architecture that allows the library to be non-commital to the state management system, ultimately allowing the transition from Flux to Redux to be a much more incremental process.
+
+### Should you implement an Influx architecture during the transition?
+__tl;dr__
+
+__If you assess your architecture and are confident you can go straight from Flux to Redux, you should not use Influx. If you are not confident, you should consider it.__
+
+The Influx architecture is not a required part of the process, and may make your life more difficult by adding extra steps. There are both advantages and disadvantages, and assessing how large the refactor is will likely provide the largest indication of whether or not it makes sense.
+
+### Advantages
+- __You can split the effort into very tangible subtasks.__ Stores can be updated to Redux without necessarily updating the Flux components. Components reliant on multiple stores can talk to Flux stores and Redux stores at the same time. Ultimately, this means a library can pick and choose the areas that can be updated and do them one at a time.
+- __Inability to update entirely to `Component2` is not a blocker.__ Since `connected` components need to be `Component2`, the Redux refactor may be blocked by those efforts. This option allows the library to update the state system to a Redux friendly architecture while updating components to `UiComponent2` and `connect` at the same time.
```

Comment:
```suggestion
- __Inability to update entirely to `Component2` is not a blocker.__ Since `connected` components need to be `Component2`, the Redux refactor may be blocked by those efforts. This option allows the library to update the state system to a Redux friendly architecture while [updating components to `UiComponent2`](./ui_component2_transition.md) and `connect` at the same time.
```